parser.go

```diff
@@ -18,6 +18,7 @@ const (
 	coloptBinary
 	coloptCharacterSet
 	coloptCollate
+	coloptValues
```

Comment:
I have not looked it up, but is "Values" the proper term used in the MySQL documentation for Set/Enums? If not, the term seems a bit too generic.

Reply:
I agree.